index.js

```diff
@@ -0,0 +1,7 @@
+const PostgresErrorConverter = require('./src/PostgresErrorConverter.js');
+
+module.exports = function(options) {
```

Comment:
Could you please expose the class itself?

Reply:
Would you only expose the class or both the register function and the class?